### PR TITLE
New distributions

### PR DESCRIPTION
**Pull request by Lesh79**
Added new distributions:
- Exponential
- Weibull
- Uniform
- Beta

### CODE REVIEW

**Review by vkutuev — DISMISSED** [review]

File: CPDShell/generator/distributions.py [hunk 1]
```diff
@@ -97,3 +109,181 @@ def from_params(params: dict[str, str]) -> "NormalDistribution":
...
+        }
+
+    def scipy_sample(self, length: int) -> np.ndarray:
+        return ss.expon.rvs(scale=1 / self.rate, size=length)
+
+    @staticmethod
+    def from_params(params: dict[str, str]) -> "ExponentialDistribution":
+        if len(params) != 1:
+            raise ValueError(
+                "Exponential distribution must have 1 parameters: " + f"{ExponentialDistribution.RATE_KEY}"
```
Comment (alexdtat) [inline CPDShell/generator/distributions.py]: 1 parameter

File: tests/test_CPDShell/test_configs/test_config_exp.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+- name: exp
+  all_distributions:
```
Comment (alexdtat) [inline tests/test_CPDShell/test_configs/test_config_exp.yml]: I briefly checked config_parser.py, shouldn't it be just `distributions`?

File: CPDShell/main.py [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+from CPDShell.shell import CPDShell
 
-scenario = Scenario(10, True)
-scrubber = Scrubber(scenario, [1, 1.2, 12, 13, 11, 12, 1, 1, 1, 1, 1])
-arg = 5
-core = CPDCore(scrubber, GraphAlgorithm(lambda a, b: abs(a - b) <= arg, 2))
+with tempfile.TemporaryDirectory() as tempdir:
+    saver = DatasetSaver(Path(), True)
+    generated = ScipyDatasetGenerator().generate_datasets(
+        Path("tests/test_CPDShell/test_configs/test_config_exp.yml"), saver
```
Comment (alexdtat) [inline CPDShell/main.py:11]: For some reasons it raises "Incorrect config path" error when I'm trying to execute it.

**Review by alexdtat — CHANGES_REQUESTED** [review]
I'm not sure where is the problem, but check it please once more.

File: CPDShell/main.py (hunk 3 above)
Comment (Lesh79) [reply CPDShell/main.py:11]: Start it up main.py from the root folder

**Review by alexdtat — APPROVED** [review]
LGTM

**Pull request merged**